biogeophys/FatesPlantHydraulicsMod.F90

```diff
@@ -1522,7 +1606,10 @@ subroutine RecruitWUptake(nsites,sites,bc_in,dtime,recruitflag)
     ! After the root water uptake, is_newly_recruited flag is set to false.
     ! Note, this routine is not accounting for the normal water uptake of new plants
     ! going forward, this routine accounts for the water that needs to be accounted for
-    ! as the plants pop into existance.    
+    ! as the plants pop into existance.  
+    ! Notes by Junyan,  July 16. 2020
+    !   modify the accessable soil layer equal to z_fr_0
```

Comment:
Is that still tbd?

Reply:
As far as I can tell, this should be implicitly true, since rooting depth has already informed each cohorts array of fine absorbing roots with depth: ccohort_hydr%v_aroot_layer(:)